arch/arm/core/aarch64/mmu/arm_mmu.c

```diff
@@ -498,3 +610,12 @@ void arch_mem_map(void *virt, uintptr_t phys, size_t size, uint32_t flags)
 		k_panic();
 	}
 }
+
+void arch_mem_unmap(void *addr, size_t size)
+{
+	int ret = remove_map(&kernel_ptables, "generic", (uintptr_t)addr, size);
+
```

Comment:
needs `TLBI` . 
`arch_mem_map`  needs them too.
and the `enable_mmu_el1`  as well.